Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py

```diff
@@ -455,7 +474,7 @@ def get_details_of_an_abuse_mailbox_campaign_command(client, args):
 
     response = client.get_details_of_an_abuse_mailbox_campaign_request(campaign_id, subtenant)
     command_results = CommandResults(
-        outputs_prefix='AbnormalSecurity.AbuseCampaign.campaigns',
+        outputs_prefix='AbnormalSecurity.AbuseCampaign',
```

Comment:
why changing the outputs_prefix? 
```suggestion
        outputs_prefix='AbnormalSecurity.AbuseCampaign.campaigns',
```